hod/subcommands/connect.py

```diff
@@ -61,38 +61,39 @@ class ConnectSubCommand(SubCommand):
     """
     CMD = 'connect'
     HELP = "Connect to a hod cluster."
-    EXAMPLE = "hod connect <jobid>"
+    EXAMPLE = "hod connect <label>"
 
     def run(self, args):
         """Run 'connect' subcommand."""
         optparser = ConnectOptions(go_args=args, envvar_prefix=self.envvar_prefix, usage=self.usage_txt)
         try:
             if len(optparser.args) > 1:
-                cluster_label = optparser.args[1]
+                label = optparser.args[1]
             else:
-                sys.stderr.write("No jobid provided.\n")
-                return 1
+                _log.error("No label provided.")
+                sys.exit(1)
```

Comment:
The return code from this function will be the exit code for the process. If you change this here, please change the tests to use `assertRaiseRegexp(SystemExit, '1', ...)` when calling run.